src/streamlink/plugins/blasttv.py

```diff
@@ -0,0 +1,74 @@
+
+
+@pluginmatcher(
+    re.compile(
+        r"^https?://(www\.)?blast\.tv/live(/(?P<channel>[a-z0-9_-]+))?$",
+    ),
+)
+class BlastTv(Plugin):
+
+    def _get_streams(self):
+        channel = self.match.group("channel")
+
+        live_channels = self.session.http.get(
+            "https://api.blast.tv/v1/broadcasts/live",
+            acceptable_status=(200,),
+            schema=validate.Schema(
+                validate.parse_json(),
+                validate.filter(lambda v: v.get("videoSrc")),
+                [
+                    {
+                        "id": str,
+                        "slug": str,
+                        "title": str,
+                        "videoSrc": validate.url(),
+                    },
+                ],
+            ),
```

Comment:
Parsed JSON responses require a proper data structure before you can iterate the data (validate.filter). If they don't return an array, then this will raise an ugly error.